### PR TITLE
Add last_known_event_id property

### PR DESCRIPTION
**Pull request by andybalaam**
As agreed here: https://github.com/matrix-org/matrix-spec-proposals/pull/3946#discussion_r1090502524

### CODE REVIEW

File: proposals/3946-dynamic-predecessor.md [hunk 1]
```diff
@@ -49,6 +49,7 @@ key | type | value | description | required
 --- | --- | --- | --- | ---
 `predecessor_room_id` | string | Room ID | A reference to the room that came before and was replaced by this room | yes
 `via_servers` | [string] | List of server names | The servers to attempt to join the room through. These servers should be participating in the room to be useful. This option is necessary since room ID's are not routable on their own.
+`last_known_event_id` | string | Event ID | The event ID of the last known event in the old room
```
Comment (MadLittleMods) [inline proposals/3946-dynamic-predecessor.md:52]: Why is this useful? We should continue the conversation in https://github.com/matrix-org/matrix-spec-proposals/pull/3946#discussion_r1086423150

**Comment by turt2live** [comment]
(both of you should also have push permissions on the MSC, and this seems small enough to fit into the "suggestions" feature on github: it's not cleart why this PR exists)

File: proposals/3946-dynamic-predecessor.md (hunk 1 above)
Comment (andybalaam) [reply proposals/3946-dynamic-predecessor.md:52]: > (both of you should also have push permissions on the MSC, and this seems small enough to fit into the "suggestions" feature on github: it's not cleart why this PR exists)

Yeah I probably should have used suggestions - I didn't think of that, sorry.

**Comment by turt2live** [comment]
Closing in favour of further discussion on the MSC itself.

**Pull request closed without merging**